### PR TITLE
fix(ci): Add debug output to in order to make it easier to  investiga…

### LINKED ISSUE #14294: eventd is correctly started for the integration tests
Currently it does not start on master because ntpdate cannot get the time from an ntp server.

Added debug logic in this pr https://github.com/magma/magma/pull/14238

First runs to include these changes are: https://github.com/magma/magma/actions/runs/3336178178 and  https://github.com/magma/magma/actions/runs/3335602125

### PR DESCRIPTION
**Pull request by crasu**
Add debug output to in order to make it easier to investigate services not starting in integ tests

## Summary

There have been several instances when eventd did not startup during agw integ tests during the last days. As this does not happen locally this will enable us to investigate these issues further. Currently I can only reproduce this on master ;-(

Example:
[1](https://storage.googleapis.com/magma-ci.appspot.com/lte_integ_test_67098eeadbaf786f2c4de06eaec3b8fd91fddb5b.html) [2](https://storage.googleapis.com/magma-ci.appspot.com/lte_integ_test_ab2e8655a74cfb7d9ec98942712d923c57a24fc5.html) [3](https://storage.googleapis.com/magma-ci.appspot.com/lte_integ_test_943e63c115fcebda04d2638188d49d83b14024ff.html) [4](https://storage.googleapis.com/magma-ci.appspot.com/lte_integ_test_6861ffa729c3b7c347be9d02308f6111a94a1309.html)

## Integ Run

https://github.com/crasu/magma/actions/runs/3300444273 (green run ;-(
https://github.com/crasu/magma/actions/runs/3320152230 (some flaky test failures)

### CODE REVIEW

File: lte/gateway/configs/health.yml [hunk 1]
```diff
@@ -22,6 +22,6 @@ state_recovery:
 
   # Number of restarts of services_check that triggers recovery process
   restart_threshold: 2
-  interval_check_mins: 3
+  interval_check_mins: 1
```
Comment (nstng) [inline lte/gateway/configs/health.yml]: Why this change? The PR description lists systemd issues.

File: lte/gateway/python/integ_tests/s1aptests/test_services_are_running.py [hunk 2]
```diff
@@ -105,6 +106,11 @@ def _query_state_of_services(self, service_status):
             print(f'  {active_state}')
             print(f'  {start_time}')
 
+    def get_failed_service_info(self, failed_service):
```
Comment (nstng) [inline lte/gateway/python/integ_tests/s1aptests/test_services_are_running.py:109]: Currently this tests only runs for a systemd setup. But if TBD (PR from @mpfirrmann) gets merged in its current state then calling this function does not really give a helpful output. I assume the `errors='ignore'` will just cause an output of `Unit eventd.service could not be found.`.

File: lte/gateway/configs/health.yml (hunk 1 above)
Comment (crasu) [reply lte/gateway/configs/health.yml]: Can be removed no idea how that slipped in here

**Comment by crasu** [comment]
Looks like the issues were caused by ntp. Syslog contains the following message:
`Oct 19 19:24:47 magma-dev eventd[67686]: 19 Oct 19:24:47 ntpdate[67686]: no server suitable for synchronization found`

File: lte/gateway/deploy/roles/magma/files/systemd/magma_eventd.service [hunk 3]
```diff
@@ -15,7 +15,7 @@ Description=Magma eventd service
 [Service]
 Type=simple
 EnvironmentFile=/etc/environment
-ExecStartPre=/usr/sbin/ntpdate pool.ntp.org
+ExecStartPre=/usr/sbin/ntpdate -vd pool.ntp.org
```
Comment (crasu) [inline lte/gateway/deploy/roles/magma/files/systemd/magma_eventd.service:18]: should give us some debug output on master.

File: lte/gateway/python/integ_tests/s1aptests/test_services_are_running.py (hunk 2 above)
Comment (crasu) [reply lte/gateway/python/integ_tests/s1aptests/test_services_are_running.py:109]: For the docker case we probably would need a docker logs <something>. Once this pr exists/gets merged.

**Review by sebathomas — APPROVED** [review]

**Review by mpfirrmann — APPROVED** [review]
